admin/tools/cluster/cc-in2p3/shmux/hosts.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+BEG=100
+END=124
+
+# ccqserv100.in2p3.fr is the master
+# and xrootd start faster is master is started last
+for i in $(seq $BEG $END| tac); do
+    printf "ccqserv${i}.in2p3.fr\n"
```

Comment:
With current format `echo` will work fine, or did you want to write `printf "ccqserv%03d.in2p3.fr\n" $i`?